src/main/java/org/opensearch/flowframework/common/FlowFrameworkSettings.java

```diff
@@ -44,4 +44,14 @@ private FlowFrameworkSettings() {}
         Setting.Property.NodeScope,
         Setting.Property.Dynamic
     );
+
+    /** This setting sets the maximum number of transport request retries */
+    public static final Setting<Integer> MAX_REQUEST_RETRY = Setting.intSetting(
```

Comment:
I think the name and javadoc is too broad.  We're applying this to task retries, not all transport requests.

Reply:
Sure ill make this name change